next.config.js

```diff
@@ -43,7 +42,7 @@ module.exports = {
       },
       {
         source: '/organize/:orgId(\\d{1,})/areas',
-        destination: '/legacy?path=/maps&orgId=:orgId',
+        destination: `https://organize.${process.env.ZETKIN_API_DOMAIN}/maps&orgId=:orgId`,
```

Comment:
Correct me if I'm wrong, but I believe there's a problem here, because this takes away the warning page that was there before.

Before, /organize/1/areas was an **internal** redirect to a warning page saying "You are being redirected". But now, I believe it just redirects directly. I think what you need to do in order to fix this is to revert this to what it was before (**internal**) but change the `legacy.tsx` page to link (**externally**) to the correct instance.

Reply:
That's right! I'll fix it asap